src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -915,5 +918,82 @@ void AppHost::_HandleSettingsChanged(const winrt::Windows::Foundation::IInspecta
 void AppHost::_IsQuakeWindowChanged(const winrt::Windows::Foundation::IInspectable&,
                                     const winrt::Windows::Foundation::IInspectable&)
 {
+    if (_window->IsQuakeWindow() && !_logic.IsQuakeWindow())
+    {
+        // If we're exiting quake mode, we should make our
+        // tray icon disappear.
+        if (_trayIconData)
+        {
+            Shell_NotifyIcon(NIM_DELETE, &_trayIconData.value());
+            _trayIconData.reset();
+        }
+    }
+
     _window->IsQuakeWindow(_logic.IsQuakeWindow());
 }
+
+void AppHost::_HandleTrayIconPressed()
+{
+    // Currently scoping "minimize to tray" to only
+    // the quake window.
+    if (_logic.IsQuakeWindow())
+    {
+        _window->SummonWindow({});

```

Comment:
we should probably actually fill this with `DropdownDuration(200)`, so that pressing the tray icon will summon the window with the quake dropdown.

Reply:
oh yeah good catch